src/Umbraco.Web.UI.Client/src/common/directives/components/overlays/umboverlay.directive.js

```diff
@@ -260,10 +260,22 @@ Opens an overlay to show a custom YSOD. </br>
             }
 
             function setView() {
-
                 if (scope.view) {
 
-                    if (scope.view.indexOf(".html") === -1) {
+                    // if the view value has a space, treat it as an inline value
+                    // since setting the view by alias will not have a space
+                    if (scope.view.indexOf(' ') > -1) {
```

Comment:
@nathanwoulfe couldn't this potential break stuff if developers have specified a view like `~/App_Plugins/My Plugin/view.html`?

Maybe it is fine to assume at least any html it added, e.g. it contains at least `<` and `>`?
Does it still work using the shorthand view?

Reply:
@bjarnef good point - I could check for spaces and the absence of `.html`, although that would fail if core added/changed existing overlay naming to have spaces. Maybe require the literal view be wrapped in an HTML element, and can then checks `startsWith('<')` and `endsWith('>')`. It probably should require HTML rather than a bare string, in the interest of semantic HTML and to be consistent with Angular directive templates, which require a single root element.

I don't want to introduce a new property for the literal view, really want to manage it in code - cleaner and friendly for developers...